mods/stairs/init.lua

```diff
@@ -855,3 +861,60 @@ stairs.register_stair_and_slab(
 	default.node_sound_snow_defaults(),
 	true
 )
+
+stairs.register_stair_and_slab(
+	"glass",
+	"default:glass",
+	{cracky = 3, oddly_breakable_by_hand = 3},
+	{"default_glass.png", "default_glass.png", "stairs_glass_split.png"},
+	"Glass Stair",
+	"Glass Slab",
+	default.node_sound_glass_defaults(),
+	false
+)
+
+stairs.register_stair(
+	"obsidian_glass",
+	"default:obsidian_glass",
+	{cracky = 3},
+	{"stairs_obsidian_glass_split.png", "default_obsidian_glass.png",
+	"stairs_obsidian_glass_outer_stairside.png", "stairs_obsidian_glass_outer_stairside.png",
```

Comment:
Should be: `"stairs_obsidian_glass_stairside.png^[transformFX", "stairs_obsidian_glass_stairside.png",`